models/articles.models.js

```diff
@@ -37,3 +37,19 @@ exports.selectAllCommentsByArticleId = (article_id) => {
     return result.rows;
   });
 };
+
+exports.updateArticleById = (article_id, { inc_votes }) => {
+  return this.selectArticleById(article_id).then(() => {
```

Comment:
You might not need to check the article exists first as PSQL should generate some errors for you if you try and update an article that doesn't exist however your solution is absolutely fine to do.

Just as a note, in production making two database calls every time may be costly however there'd definitely be analytics to see whether this extra step is benificial.

Reply:
Thanks Lewis.

I take your point regarding the 2 database calls and have refactored to using a single query and let the database do the deciding.  😄